packages/odyssey-react/src/components/Icon/SvgIcon.tsx

```diff
@@ -36,28 +42,17 @@ export interface SvgIconProps
 
 let SvgIcon = forwardRef<SVGSVGElement, SvgIconProps>(
   ({ title, children, ...rest }, ref) => {
-    const autoId = "icon_" + useOid();
     const omitProps = useOmit(rest);
 
     return Children.only(
-      cloneElement(
-        children,
-        {
-          ...omitProps,
-          "aria-labelledby": title && autoId,
-          className: styles.root,
-          ref: ref,
-          role: title ? "img" : "presentation",
```

Comment:
Either will work but "presentaion" feels more correct in this case and is (despite the storybook error) a valid role

Reply:
I don't believe the "img" role needs to be set explicitly, if not specified it should fall back to its default of "graphics-document" which should be sufficient